app/data/tasks/deduplicate_records.py

```diff
@@ -0,0 +1,185 @@
+from django.conf import settings
+from django.core.paginator import Paginator
+
+from celery import shared_task
+from celery.utils.log import get_task_logger
+
+from data.models import (DedupeJob, RecordDuplicate)
+from ashlar.models import Record
+
+import datetime
+from pytz import timezone
+
+logger = get_task_logger(__name__)
+
+
+def get_time_allowance():
+    return datetime.timedelta(hours=settings.DEDUPE_TIME_RANGE_HOURS)
+
+
+# .001 degrees is ~110m give or take 10 depending on location, which should
+# be good enough
+def get_distance_allowance():
+    return settings.DEDUPE_DISTANCE_DEGREES
+
+
+def calculate_score(record, duplicate):
```

Comment:
I'd change this to `calculate_similarity_score` so it's clear what it does -- the `duplicate` parameter could perhaps also be renamed since at this point we're not sure it's a duplicate yet.